NETCORE/src/Shared/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -170,7 +170,7 @@ internal void CopyPropertiesTo(ApplicationInsightsServiceOptions target)
                 target.InstrumentationKey = this.InstrumentationKey;
             }
 
-            target.ConnectionString = this.ConnectionString;
+            target.ConnectionString = this.ConnectionString ?? target.ConnectionString;
```

Comment:
i would use the same pattern above.... if not null or empty

Reply:
Done.